.github/file-paths.yaml

```diff
@@ -84,8 +84,8 @@ snowplow:
 
 documentation:
   - "docs/**"
-  - "**.md"
+  - "**/*.md"
 
 yaml:
-  - "**.yml"
-  - "**.yaml"
+  - "**/*.yml"
+  - "**/*.yaml"
```

Comment:
The same principle applies here.
https://github.com/metabase/metabase/pull/28447/files#r1111985035